### PR TITLE
Grammar correction

### PR DESCRIPTION
**Pull request by abhi1604**
Made a small gramatical correction to a sentence.

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ While `--apiserver-advertise-address` can be used to set the advertise address f
 control-plane node's API server, `--control-plane-endpoint` can be used to set the shared endpoint
 for all control-plane nodes.
 
-`--control-plane-endpoint` allows IP addresses but also DNS names that can map to IP addresses.
+`--control-plane-endpoint` allows not only IP addresses but also DNS names that can map to IP addresses.
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:121]: this is valid as per the "bot only … but also" sequence, but it think it makes the sentence a bit too pompous?

how about changing it to:
> `--control-plane-endpoint` allows both IP addresses and DNS names that can map to IP addresses.

/sig cluster-lifecycle
/kind cleanup

Comment (abhi1604) [reply content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:121]: yeah, this is easier to understand.

**Comment by neolit123** [comment]
thanks, please squash the two commits to one.

**Pull request merged**